Assets/Scripts/TextureCache.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System.Collections.Generic;
+using System.IO;
+using Assets.Scripts.Model;
+using UnityEngine;
+
+public class TextureCache : MonoBehaviour
+{
+
+  const string c_defaultTexturePath = "Textures/node/sand_01_diff_1k.jpg";
+
+  public IDictionary<int, Material> cache = new Dictionary<int, Material>();
+
+  public void Load(IEnumerable<NodeClass> classes)
+  {
+    foreach (var nodeClass in classes)
+    {
+      var texture = new Texture2D(2, 2);
```

Comment:
what are the 2s?

Reply:
It's supposed to be width/height but I have not found what thos changes, I think it is overriden by loaded texture. I will look into it some more.

Reply:
sure, I would like those extracted as consts